_partials/_migrate_dual_write_step2.md

```diff
@@ -7,8 +7,20 @@ you must think about how to handle the failure to write to either the source or
 target database, and what mechanism you want to or can build to recover from
 such a failure.
 
+Should your time-series data have foreign-key references into a plain table,
+you must ensure that your application correctly maintains the foreign key
+relations. Beware that if the referenced column is a `*SERIAL` type, the same
+row inserted into the source and target _may not_ obtain the same autogenerated
+id. If this happens, the data backfilled from the source to the target will be
+internally corrupt. In the best case it causes a foreign key violation, in the
+worst case, the foreign key relation constraint is maintained, but the data
+references the wrong foreign key. If reading this has made you worried,
+consider using [live migration] instead.
```

Comment:
If reading this has made you worried, consider using [live migration] instead.

To avoid these issues, best practice is to follow [live migration][live-migration].

Reply:
done